components/dashboard/src/projects/NewProject.tsx

```diff
@@ -45,6 +45,14 @@ export default function NewProject() {
                 setProvider("github.com");
             }
         }
+        if (user) {
+            if (!user?.rolesOrPermissions?.includes('teams-and-projects')) {
+                (async () => {
+                    setUser(await getGitpodService().server.getLoggedInUser());
+                })();
+
+            }
+        }
```

Comment:
You could simplify this Effect's 🍝 code a bit by adding `if (!user) { return; }` at the top. (Also the empty line is unnecessary. 😇)

Reply:
I'd love to see that PR reverted rather soon, i.e. getting to general availability of Teams & Projects. I would be great to just revert this PR by then with less effort. WDYT?